wagtail_localize_rws_languagecloud/tests/test_views.py

```diff
@@ -0,0 +1,115 @@
+from django.test import TestCase
+from django.test.utils import override_settings
+from django.urls import reverse
+from django.utils import timezone
+from wagtail.core.models import Locale
+
+from wagtail_localize.models import Translation
+
+from ..models import LanguageCloudFile, LanguageCloudProject, LanguageCloudStatus
+from .helpers import create_editor_user, create_test_page
+
+
+@override_settings(
+    WAGTAILLOCALIZE_RWS_LANGUAGECLOUD={
+        "CLIENT_ID": "fakeid",
+        "CLIENT_SECRET": "fakesecret",
+        "ACCOUNT_ID": "fakeaccount",
+    },
+)
```

Comment:
I might be missing something here, but why do we need to mock the settings for these tests?

Reply:
Somewhere in the call stack when rendering the "Sync translations" page, the API client is getting initialized. I believe it's when the custom RWS LanguageCloud component is rendered. Let me check exactly where.

Reply:
That is coming from `wagtail_localize_rws_languagecloud/forms.py`

Reply:
Ah yes - here
https://github.com/zerolab/wagtail-localize-rws-languagecloud/blob/5b3cd2577b9f1a66e1608b95ee9ba664e6c77d63/wagtail_localize_rws_languagecloud/forms.py#L127
I don't think I have a strong preference for whether we mock the settings or the method. I just wasn't sure why we needed to hit the API in this code path.

Reply:
Leaving this as is.